lib/providers/ModuleApiProvider.js

```diff
@@ -24,6 +24,7 @@ class ModuleApiProvider extends ModuleApiProviderBase {
 		 */
 		this.actions = {
 			redirectedTo: '',
+			errorStatusCode: null,
```

Comment:
Also, would be nice to make `notFound` an alias for `error(404)` and `redirect` to set `statusCode: 3xx`